V1.01_ROM_disassembly_C000.asm

```diff
@@ -334,403 +337,455 @@ label_c19b:
     ex      (sp),hl                         ;[c19b] e3
     ret                                     ;[c19c] c9
 
-    push    bc                              ;[c19d] c5
-    push    de                              ;[c19e] d5
-    push    hl                              ;[c19f] e5
-    ld      ($ffb8),a                       ;[c1a0] 32 b8 ff
-    ld      a,$0a                           ;[c1a3] 3e 0a
-    ld      ($ffbf),a                       ;[c1a5] 32 bf ff
-    ld      ($ffb9),bc                      ;[c1a8] ed 43 b9 ff
-    ld      ($ffbb),de                      ;[c1ac] ed 53 bb ff
-    ld      ($ffbd),hl                      ;[c1b0] 22 bd ff
-    call    $c423                           ;[c1b3] cd 23 c4
-    ld      a,($ffba)                       ;[c1b6] 3a ba ff
-    and     $f0                             ;[c1b9] e6 f0
-    jp      z,$c1e6                         ;[c1bb] ca e6 c1
-    cp      $40                             ;[c1be] fe 40
-    jp      z,$c1dc                         ;[c1c0] ca dc c1
-    cp      $80                             ;[c1c3] fe 80
-    jp      z,$c1d7                         ;[c1c5] ca d7 c1
-    cp      $20                             ;[c1c8] fe 20
-    jp      z,$c1e1                         ;[c1ca] ca e1 c1
-    cp      $f0                             ;[c1cd] fe f0
-    jp      z,$c1eb                         ;[c1cf] ca eb c1
-    ld      a,$ff                           ;[c1d2] 3e ff
-    jp      $c1f0                           ;[c1d4] c3 f0 c1
-    call    $c1f4                           ;[c1d7] cd f4 c1
-    jr      label_c1f0                      ;[c1da] 18 14
-    call    $c24a                           ;[c1dc] cd 4a c2
-    jr      label_c1f0                      ;[c1df] 18 0f
-    call    $c3a9                           ;[c1e1] cd a9 c3
-    jr      label_c1f0                      ;[c1e4] 18 0a
-    call    $c391                           ;[c1e6] cd 91 c3
-    jr      label_c1f0                      ;[c1e9] 18 05
-    call    $c2e3                           ;[c1eb] cd e3 c2
-    jr      label_c1f0                      ;[c1ee] 18 00
-label_c1f0:
+    ; FDC Read Write Format Seek routine.
+    ; Arguments:
+    ; a: TODO
+    ; b: drive number (0-3) + HD flag
+    ; c: operation command, see switch in this routine
+    ; d: track number
+    ; e: head number
+    ; hl: read/write buffer address
+fdc_rwfs_c19d:
+    push    bc                              ;[c19d]
+    push    de                              ;[c19e]
+    push    hl                              ;[c19f]
+    ld      ($ffb8),a                       ;[c1a0] TODO
+    ld      a,$0a                           ;[c1a3]
+    ld      ($ffbf),a                       ;[c1a5] value used in error checking routines
+    ld      ($ffb9),bc                      ;[c1a8] *$ffb9 = drive no. + HD flag, *$ffba = operation command
```

Comment:
This stores C register in `$ffb9` and B register in `$ffba`.
Then, at line 359, value from variable at `$ffb9` is used in the switch case.
Then, I think that B originally contains the operation command, and C the drive number.
That's also what you commented in `bios_bootkey` when this routine is called.
(Just leaving this as a comment, can't verify right now)